sklearn/feature_selection/_mutual_info.py

```diff
@@ -225,11 +225,9 @@ def _estimate_mi(X, y, discrete_features='auto', discrete_target=False,
 
     random_state : int, RandomState instance or None, optional, default None
         The seed of the pseudo random number generator for adding small noise
```

Comment:
It's not actually a seed since it can be a RandomState instance. I'd say "Determines random number generation for adding...". wdyt ?

Reply:
I addressed your comment @jeremiedbb